src/lib/parseFilters.ts

```diff
@@ -8,13 +8,14 @@ import {
   filter__post_getPostSummaryByPids,
   filter__teasers_get,
 } from './hooks';
+import { getResponsesCount } from './responses';
 
 const eventRX = new RegExp(tagTemplate('event', '[\\s\\S]*'));
 const invalidRX = new RegExp(`(${tagTemplate('event-invalid', '[\\s\\S]*')})`);
 
 const reescape = (str: string) => validator.escape(validator.unescape(str));
 
-const parseRaw: filter__parse_raw = async (content) => {
+const parseRawWithPid = async (content: string, pid?: number) => {
```

Comment:
Not sure if this is the right approach, but I don't see any other way how to keep the code DRY and keep 'parseRaw' working. Without using dynamic Sockets.

Reply:
This is fine, but I'd prefer a different name. `parseContent` maybe?